app/src/main/java/com/example/androidtrainingproject/products/ProductDetailsScreen.kt

```diff
@@ -66,46 +83,125 @@ fun ProductDetails() {
+                                color = Red
+                            )
+                        }
+                        
+                    }
+                }
+                Row(
+                    modifier = Modifier.padding(top = dimensionResource(id = R.dimen.padding_sm_plus)),
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Text(
+                        text = product.title,
+                        fontWeight = FontWeight.Bold
+                    )
+                    Spacer(modifier = Modifier.weight(0.5f))
+                    Text(text = product.rating.toString())
+                    for (i in 0 until product.rating.toInt()) {
+                        Icon(
+                            modifier = Modifier.height(dimensionResource(id = R.dimen.icon_small)),
+                            imageVector = Icons.Outlined.Star,
+                            contentDescription = null,
+                            tint = Purple
+                        )
+                    }
+                    for (i in product.rating.toInt() until 5) {
+                        Image(
+                            modifier = Modifier.height(dimensionResource(id = R.dimen.icon_small)),
+                            painter = painterResource(id = R.drawable.rating_star_outlined),
+                            contentDescription = null
+                        )
+                    }
+                }
+                Text(
+                    text = stringResource(id = R.string.product_category_label) + " " + product.category,
+                    fontSize = dimensionResource(id = R.dimen.font_size_sm).value.sp,
+                    color = LabelText
                 )
                 Text(
-                    modifier = Modifier.padding(bottom = 10.dp),
+                    modifier = Modifier
+                        .padding(
+                            top = dimensionResource(id = R.dimen.padding_m),
+                            bottom = dimensionResource(id = R.dimen.padding_sm)
+                        ),
                     text = product.description,
-                    fontSize = 16.sp
+                    fontSize = dimensionResource(id = R.dimen.font_size_m).value.sp
                 )
                 Text(
                     text = "$${String.format("%.2f", product.price)}",
-                    fontSize = 30.sp,
+                    fontSize = dimensionResource(id = R.dimen.font_size_xl).value.sp,
                     fontWeight = FontWeight.Bold
                 )
-            }
-            Button(
-                modifier = Modifier
-                    .padding(top = 50.dp)
-                    .padding(horizontal = 20.dp)
-                    .fillMaxWidth(),
-                onClick = {},
-                colors = ButtonDefaults.buttonColors(containerColor = Color(0xFF47337A))
-            ) {
-                Text("Add to cart")
+                Button(
```

Comment:
Here you can use your shared reusable `Button`, parametrized in a way where you can achieve both looks.